### DISCUSSION TITLE
How to upgrade the releases version?

### DISCUSSION THREAD

**Discussion by ChoiChimta** [Zibbp/ganymede#270] [Q&A] [answered]
Hi, 
I'm sorry I'm using a translator.

Is a clean installation of ganymede the only way to upgrade releases version?
I looked for a similar question, but I couldn't find it

If so, the files downloaded to the old version of Ganymede will remain on the NAS but will not be displayed on the frontend.
Is there a way to automatically upload to a new version of ganymede?

**Comment by Zibbp** [comment, marked as answer]
If you are using the docker images with the `:latest` tag you just need to pull the latest images `docker compose pull` and `docker compose up -d`. This will update to the latest released version.

**Reply by ChoiChimta** [reply]
WOW!
I installed a new one last time, and it was stupid.
THX 👍
